CONTRIBUTING.md

```diff
@@ -26,6 +27,100 @@ The easier it is for us to reproduce it, the faster it'll be fixed!
 
 Please don't include any private/sensitive information in your issue!
 
+## Hack On Libpod
+
+This section describe to you how to start to hack on libpod.
+
+### Prepare your environment
+
+Read the [install documentation to see how to install dependencies](install.md) .
+
+During the reading of the install documentation we will as to you to:
```

Comment:
s/as to you/ask you/

and really I'd prefer:

"The install documentation will illustrate the following steps:"